src/orderbook.js

```diff
@@ -164,7 +201,15 @@ OrderBook.EVENTS = [
   'offer_changed', 'offer_funds_changed'
 ];
 
-OrderBook.DEFAULT_TRANSFER_RATE = 1000000000;
+OrderBook.DEFAULT_TRANSFER_RATE = Amount.from_json(1000000000);
+
+OrderBook.NOTIFY_THROTTLE_TIME = 4000;
```

Comment:
under heavy load, the debounce wouldn't get a sufficient pause, so the the # of calls to  `compute` would be limited to the `maxWait` interval which would be at 500 ms - we could have a system that dynamically increases the maxWait if the compute is taking too long.